src/memex/search/parser.py

```diff
@@ -76,6 +76,31 @@ def parse(q):
     return MultiDict([m for m in parse_results if isinstance(m, Match)])
 
 
+def unparse(q):
+    """
+    Turn a dict-like object into a Lucene-like query string.
+
+    This can be considered the reverse of the
+    :py:func:`memex.search.parser.parse` function, as it can be used to
+    transform the MultiDict returned from that function back into a string
+    query.
+    """
+    terms = []
+
+    try:
+        items = q.iteritems(multi=True)
```

Comment:
The `webob.multidict.MultiDict` object returned by `memex.search.parser.parse` does not support a 'multi' keyword argument. It looks like that is a feature of `werkzeug.datastructures.MultiDict`. Is there any reason that we would be using that type with this function?